homeassistant/components/sensor/darksky.py

```diff
@@ -97,6 +98,14 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         _LOGGER.error("Latitude or longitude not set in Home Assistant config")
         return False
 
+    # Check to make sure that forecast is in a valid range of 1 week
+    forecast = config.get(CONF_FORECAST)
```

Comment:
You'll have to nest it inside the `vol` list type. There should be some examples in other components.